### PR TITLE
Add Deprecation Version to @Deprecation tags

### PR DESCRIPTION
**Pull request by thelooter**
# Description
Adds the Deprecation Versions to the Deprecation Tags

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Code follows existing style.
- [x] Unit tests added (if applicable).

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java [hunk 1]
```diff
@@ -160,14 +162,12 @@ public void setEatingHaystack(boolean eatingHaystack)
 	}
 
 	@Override
-	@Deprecated
```
Comment (Insprill) [inline src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java:163]: Was this mistakenly marked as deprecated or is this a mistake?

File: src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java [hunk 2]
```diff
@@ -160,14 +162,12 @@ public void setEatingHaystack(boolean eatingHaystack)
...
 	public boolean isEatingGrass()
 	{
 		return this.isEatingHaystack();
 	}
 
 	@Override
-	@Deprecated
```
Comment (Insprill) [inline src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java:170]: Same here.

File: src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java (hunk 1 above)
Comment (thelooter) [reply src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java:163]: Those were mistakenly marked as deprecated

File: src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java (hunk 2 above)
Comment (thelooter) [reply src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java:170]: Those were mistakenly marked as deprecated

**Review by Insprill — APPROVED** [review]
LGTM

**Pull request merged**